### PR TITLE
 added alert badges to alert pages and css fixes

### PR DESCRIPTION
**Pull request by L-Steinmacher**
**Description**
Added badges to the alerts that update with the backend.

**What work was done?**  
wired up the badges to the alerts and added css to display them.

**Why was this work done?**  
Stakeholder request

Please delete options that are not relevant.  
[x] New feature (non-breaking change which adds functionality)
Change Status  
[x] Completed, ready to review and merge  
Has This Been Tested  
[x] Yes
Checklist
[x] My code follows the style guidelines of this project  
[x] I have performed a self-review of my own code  
[] My code has been reviewed by at least one peer  
[] I have commented my code, particularly in hard-to-understand areas  
[] I have made corresponding changes to the documentation  
[x] My changes generate no new warnings  
[x] There are no merge conflicts

### CODE REVIEW

**Review by ashtilawat23 — APPROVED** [review]

**Review by eet349 — APPROVED** [review]
Looks good. lets clean up the commented code an console.logs for the next push. Otherwise good job

File: src/components/common/NavMenu.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ const NavMenu = props => {
   const context = useContext(UserContext);
   const adminContext = useContext(AdminContext);
 
-  const num = adminContext.memberReactions.Length;
+  const num = adminContext.memberReactions;
+  console.log(adminContext.memberReactions);
```
Comment (ryan-hamblin) [inline src/components/common/NavMenu.js:23]: No need for console.logs please remove from main.

File: src/components/common/Tab.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
-import React, { Component } from 'react';
+import React, { Component, useContext } from 'react';
 import PropTypes from 'prop-types';
+import { Badge } from 'antd';
 class Tab extends Component {
```
Comment (ryan-hamblin) [inline src/components/common/Tab.js:4]: Please add a line break between 3 and 4

Comment (ryan-hamblin) [inline src/components/common/Tab.js:4]: Most of this repos code is done using React Hooks. I'm curious about the use of Class components here?

File: src/components/common/Tabs.js [hunk 3]
```diff
@@ -17,19 +15,35 @@ export default function Tabs(props) {
   const onClickTabItem = tab => {
     setActiveTab(tab);
   };
+  /*
+   *  Comment[ here ]
```
Comment (ryan-hamblin) [inline src/components/common/Tabs.js:19]: Comment here? What is the comment? Let's not commit it unless it's actually commented.

File: src/components/pages/Alerts/Alerts.css [hunk 4]
```diff
@@ -3,16 +3,23 @@
...
+    flex-grow: inherit;
+    justify-content: center;
+    align-items: center;
     list-style: none;
-    font-size: 1.2rem;
+    font-size: 1rem;
     color: lightgray; 
     border-bottom: 4px solid lightgray;
-    padding: 0.5rem 0.75rem;
+    /* padding: 0.5rem 0.75rem; */
```
Comment (ryan-hamblin) [inline src/components/pages/Alerts/Alerts.css:22]: Let's delete unused code.

File: src/components/pages/Alerts/Alerts.css [hunk 5]
```diff
@@ -41,6 +48,25 @@
     justify-content: center;
   }
 
+  /* .badge {
```
Comment (ryan-hamblin) [inline src/components/pages/Alerts/Alerts.css:51]: please remove

File: src/components/pages/Alerts/Alerts.css [hunk 6]
```diff
@@ -41,6 +48,25 @@
...
+  @media (max-width:1650px) {
+    .tab-list {
+      flex-wrap: wrap;
+    }
+    .tab-list-item {
+      width: 20%;
+    }
+  }
+
+  /* @media (min-width:1000px) {
```
Comment (ryan-hamblin) [inline src/components/pages/Alerts/Alerts.css:68]: let's remove deleted code.

**Review by ryan-hamblin — COMMENTED** [review]
Awesome work. I did find a few things that I'd like to be changed before I approve this pull-request. Please see my inline comments. There are lots of hanging/dead code instances, the use of a class in lieu hooks, and I would like you to consider my review.

File: src/components/common/NavMenu.js (hunk 1 above)
Comment (ashtilawat23) [reply src/components/common/NavMenu.js:23]: Nice catch! On it!

File: src/components/common/Tabs.js (hunk 3 above)
Comment (ashtilawat23) [reply src/components/common/Tabs.js:19]: Taking it out now,

File: src/components/pages/Alerts/Alerts.css (hunk 5 above)
Comment (ashtilawat23) [reply src/components/pages/Alerts/Alerts.css:51]: Done, changed should be in next push

File: src/components/pages/Alerts/Alerts.css (hunk 6 above)
Comment (ashtilawat23) [reply src/components/pages/Alerts/Alerts.css:68]: Done,

File: src/components/common/Tab.js (hunk 2 above)
Comment (ryan-hamblin) [reply src/components/common/Tab.js:4]: Oh, we had actually considered using a class component in this case because...

**Pull request merged**